style-dictionary/utils/token-names.ts

```diff
@@ -51,6 +51,7 @@ export type ColorPaletteTokenName =
   | 'colorYellow100'
   | 'colorYellow600'
   | 'colorYellow700'
+  | 'colorYellow750'
```

Comment:
Where is this being used? I only see it defined here.

Reply:
Good catch - was leftover from an earlier version. Will remove